packages/suite/src/components/suite/AppNavigationPanel.tsx

```diff
@@ -9,7 +9,7 @@ const Wrapper = styled.div`
     justify-content: center;
     background: ${props => props.theme.BG_LIGHT_GREY};
     padding: 24px 32px 10px 32px;
-    z-index: 4;
```

Comment:
![Screenshot 2022-05-18 at 9 23 16](https://user-images.githubusercontent.com/33235762/168984257-d3a0e3c4-4bc3-43a3-84a6-48e3658ce1a7.png)
This has been already fixed in some issue and now it is there again.

Reply:
![Screenshot 2022-05-18 at 9 43 04](https://user-images.githubusercontent.com/33235762/168985796-8751637f-ccc2-4d33-aaad-07d1ec817872.png)

Reply:
https://user-images.githubusercontent.com/33235762/168985880-1366838e-db03-4ae1-86fa-3f1587f74f6b.mov